### PR TITLE
Auto-complete suggests 'assignees' when it should suggest 'assignee' #1022

### LINKED ISSUE #1022: Auto-complete suggests 'assignees' when it should suggest 'assignee'
When I type 'a' in filter box, I get the suggestion 'assignees' when I'm expecting to see 'assignee'

### PR DESCRIPTION
**Pull request by lejolly**
Fixes #1022

### CODE REVIEW

File: src/test/java/guitests/UIComponentsTest.java [hunk 1]
```diff
@@ -3,14 +3,30 @@
...
+        type("ass");
+        push(KeyCode.TAB);
+        FilterTextField filterTextField = find("#dummy/dummy_col0_filterTextField");
+        assertEquals("assignee", filterTextField.getText());
+    }
+
+    @Test
     public void filterTextFieldTest() {
+        doubleClick("#dummy/dummy_col0_filterTextField");
         click("#dummy/dummy_col0_filterTextField");
```
Comment (acruis) [inline src/test/java/guitests/UIComponentsTest.java]: Doesn't this `click` remove the select-all on the filter (that was caused by the previous `doubleClick`)?

Comment (lejolly) [inline src/test/java/guitests/UIComponentsTest.java]: doubleclick has never been a select all, it's select word. triple click is select all.

Comment (acruis) [inline src/test/java/guitests/UIComponentsTest.java]: Ah alright

**Comment by acruis** [comment]
Good to merge.

File: src/test/java/guitests/UIComponentsTest.java (hunk 1 above)
Comment (dariusf) [inline src/test/java/guitests/UIComponentsTest.java]: I know this is used in quite a few places, and seems to be supported across the platforms we test on, but we should eventually factor it out into some kind of select-all or triple-click method.

**Pull request merged**